### DISCUSSION TITLE
Legend Marker类型展示问题

### DISCUSSION THREAD

**Discussion by i11I04i** [antvis/G2#5992] [Q&A]
基于官方示例【双轴折线条形图】line图表配置legend({color: {itemMarker: 'line'}})，发现interval图例也被改成line。是否可以让line配置的legend只影响line本身不影响interval？
![image](https://github.com/antvis/G2/assets/58681207/1504e483-b267-41e1-929f-43eabeca5efb)

**Comment by ai-qing-hai** [comment]
@i11I04i 当前可以使用回调进行修正 
![image](https://github.com/antvis/G2/assets/65594180/aaa3ac8b-f6a7-4e34-86e5-0def324ea6cc)
